### ISSUE TITLE
Nested struct pointer is causing segmentation fault

### ISSUE THREAD

**Issue by rafaelsamenezes** [esbmc/esbmc#405]
ESBMC is generating an stack overflow at the function `goto_convert_functionst::wallop_type`
```c
struct a {
  short symbol;
} ;
struct d {
  struct a f;
};
int main(){
    struct d *g;
    g->f;
    return 0;
}
```

This may be the problem at the issue: #359

**Comment by lucasccordeiro** [comment]
Closed via 9220c432f13b31bcd61338352f39938eab077e47.

**Comment by fbrausse** [comment]
The fix has been in #1077 with #1071 being a duplicate.

**Comment by lucasccordeiro** [comment]
Thanks, @fbrausse ;-)
